AsyncDisplayKitTests/ASDisplayNodeTests.mm

```diff
@@ -1098,33 +1098,34 @@ - (void)testSubnodes
 
 - (void)testReplaceSubnodeNoView
 {
-  [self checkReplaceSubnodeWithView:NO layerBacked:NO];
+  [self checkReplaceSubnodeLoaded:NO layerBacked:NO];
 }
 
 - (void)testReplaceSubnodeNoLayer
 {
-  [self checkReplaceSubnodeWithView:NO layerBacked:YES];
+  [self checkReplaceSubnodeLoaded:NO layerBacked:YES];
 }
 
 - (void)testReplaceSubnodeView
 {
-  [self checkReplaceSubnodeWithView:YES layerBacked:NO];
+  [self checkReplaceSubnodeLoaded:YES layerBacked:NO];
 }
 
 - (void)testReplaceSubnodeLayer
 {
-  [self checkReplaceSubnodeWithView:YES layerBacked:YES];
+  [self checkReplaceSubnodeLoaded:YES layerBacked:YES];
 }
 
 
-- (void)checkReplaceSubnodeWithView:(BOOL)loaded layerBacked:(BOOL)isLayerBacked
+- (void)checkReplaceSubnodeLoaded:(BOOL)loaded layerBacked:(BOOL)isLayerBacked
 {
   DeclareNodeNamed(parent);
   DeclareNodeNamed(a);
   DeclareNodeNamed(b);
   DeclareNodeNamed(c);
+  DeclareNodeNamed(d);
 
-  for (ASDisplayNode *n in @[parent, a, b, c]) {
+  for (ASDisplayNode *n in @[parent, a, b, c, d]) {
```

Comment:
These tests forgot to set `layerBacked` on the new subnode `d` so they failed our new assertion. The method name is also super confusing so I updated it.